guides/creating-a-custom-form-field-control.md

```diff
@@ -202,7 +202,7 @@ To resolve this, remove the `NG_VALUE_ACCESSOR` provider and instead set the val
     // },
   ],
 })
-export class MyTelInput implements MatFormFieldControl<MyTel> {
+export class MyTelInput implements MatFormFieldControl<MyTel>, ControlValueAccessor {
```

Comment:
I've found only one snippet after the [ngControl](https://material.angular.io/guide/creating-a-custom-form-field-control#ngcontrol) section. It's [improving-accessibility](https://material.angular.io/guide/creating-a-custom-form-field-control#improving-accessibility).  
Probably, there's no need to add  `ControlValueAccessor` there becasue forms are not main subject of that section.

Reply:
I guess the way I've been reading the guide is that the sections build on top of each other so it would be weird if one snippet had it and the other one didn't.

Reply:
I've reread the docs and found very important point in the [errorState section](https://material.angular.io/guide/creating-a-custom-form-field-control#errorstate):

> Since we're not using an NgControl in this example, we don't need to do anything ...

So I think there's no need to add `ControlValueAccessor` in other sections except ngControl
What do you think?

Reply:
Sure, makes sense.